Util/IronPython/Lib/iptest/console_util.py

```diff
@@ -105,7 +105,7 @@ def EatToMarker(self, marker, readError=False):
     # Execute a single-line command, and return the output
     def ExecuteLine(self, line, readError=False):
         self.writer.Write(line+"\n")
-        return self.EatToPrompt(readError)[0:-2]
+        return self.EatToPrompt(readError).rstrip()
```

Comment:
Looks like it may be for the `ClrType_ipy` test (although it's disabled, probably because the script is missing).